### PR TITLE
extend localisation to include header image

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn ready for review (depends on #45 )

**Comment by miltontony** [comment]
@smn ready for review (#45 is now merged)

File: cms/models.py [hunk 1]
```diff
@@ -155,6 +155,20 @@ class Localisation(models.Model):
             'See http://www.loc.gov/standards/iso639-2/php/code_list.php '
             'for reference.'))
 
+    image = models.ImageField(
+        upload_to=lambda _, filename: 'locales/%s' % filename,
```
Comment (smn) [inline cms/models.py:159]: Are we trusting the user's file name here?

Comment (miltontony) [inline cms/models.py:159]: It get's stored in `thumbor` but never used. We always use the `uuid` that thumbor generates.

**Comment by smn** [comment]
:+1: other than the `upload_to` generation

**Pull request merged**